### DISCUSSION TITLE
沪教版化学九上第一章笔记 — scp020 的博客

### DISCUSSION THREAD

**Discussion by giscus** [scp020/scp020.github.io#3] [General]
# 沪教版化学九上第一章笔记 — scp020 的博客

第一章 开启化学之门

https://scp020.github.io/2023/10/12/chemistry-note-chapter-one/

**Comment by scp020** [comment]
LaTeX 炸掉了。

**Comment by StevehimYu** [comment]
沈阳是用沪教版吗qwq，zmn

**Reply by scp020** [reply]
是这样的。我现在在尝试配置mathjax以支持mhch宏包。。。祝我成功。

**Reply by scp020** [reply]
嗯，可以支持了

**Comment by StevehimYu** [comment]
镁带燃烧那块是Mg + O2 -> MgO，Mg打成MgO了 /ww

**Reply by StevehimYu** [reply]
第一个镁带燃烧方程式，不是第二个awa

**Reply by scp020** [reply]
谢谢。手残了。
